cartesian_impedance_advanced_controller:
    type: franka_advanced_controllers/CartesianImpedanceAdvancedController
    arm_id: panda
    joint_names:
        - panda_joint1
        - panda_joint2
        - panda_joint3
        - panda_joint4
        - panda_joint5
        - panda_joint6
        - panda_joint7
joint_impedance_advanced_controller:
    type: franka_advanced_controllers/JointImpedanceAdvancedController
    arm_id: panda
    joint_names:
        - panda_joint1
        - panda_joint2
        - panda_joint3
        - panda_joint4
        - panda_joint5
        - panda_joint6
        - panda_joint7
dual_arm_cartesian_advanced_controller:
    type: franka_advanced_controllers/DualArmCartesianImpedanceAdvancedController
    right:
        arm_id: panda_1
        robot_ip: 172.16.0.2
        joint_names:
            - panda_1_joint1
            - panda_1_joint2
            - panda_1_joint3
            - panda_1_joint4
            - panda_1_joint5
            - panda_1_joint6
            - panda_1_joint7
    left:
        arm_id: panda_2
        robot_ip: 172.16.0.3
        joint_names:
            - panda_2_joint1
            - panda_2_joint2
            - panda_2_joint3
            - panda_2_joint4
            - panda_2_joint5
            - panda_2_joint6
            - panda_2_joint7
